dipy/denoise/noise_estimate.py

```diff
@@ -58,7 +53,7 @@ def piesno(data, N, alpha=0.01, l=100, itermax=100, eps=1e-5, return_mask=False)
     sigma : float
         The estimated standard deviation of the gaussian noise.
 
-    mask (optional): ndarray
```

Comment:
OK!

I think it went away with this commit:
https://github.com/samuelstjean/dipy/commit/707a78913173cd4abf91281376ff36e6f29c50d7

Putting it back would be nice :-)

On Mon, Feb 23, 2015 at 12:04 PM, Samuel St-Jean <notifications@github.com>
wrote:

> In dipy/denoise/noise_estimate.py
> <https://github.com/arokem/dipy/pull/13#discussion_r25196654>:
>
> > @@ -58,7 +53,7 @@ def piesno(data, N, alpha=0.01, l=100, itermax=100, eps=1e-5, return_mask=False)
> >      sigma : float
> >          The estimated standard deviation of the gaussian noise.
> >
> > -    mask (optional): ndarray
>
> Suggested by elef, it's useful to sanity check if your noise estimation is
> valid by looking at the mask of identified noisy pixel, but I think people
> mistook it for a brain mask instead and where passing this mask to any
> other function requiring a mask. If you invert it, you get a bunch of
> random voxels in the background, so it's not made for segmenting the brain
> and was a source of confusion.
>
> To prevent that, it's an optional return value. No idea why the (optional)
> went away though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/arokem/dipy/pull/13/files#r25196654>.
>